core/controlplane/config/templates/cloud-config-controller

```diff
@@ -624,15 +624,12 @@ write_files:
       kubectl apply -f "${mfdir}/tiller.yaml"
 
       # Deployments
-      for manifest in {kube-dns-de,kube-dns-autoscaler-de,{{ if .Addons.ClusterAutoscaler.Enabled }}cluster-autoscaler-de,{{ end }}heapster-de{{ if .KubeResourcesAutosave.Enabled }},kube-resources-autosave{{ end }}}.yaml; do
-          kubectl apply -f "${mfdir}/$manifest"
+      for manifest in {kube-dns-de,kube-dns-autoscaler-de,kube-dashboard-de,{{ if .Addons.ClusterAutoscaler.Enabled }}cluster-autoscaler-de,{{ end }}heapster-de{{ if .KubeResourcesAutosave.Enabled }},kube-resources-autosave{{ end }}}; do
+          kubectl apply -f "${mfdir}/$manifest.yaml"
       done
 
-      # Replicationcontrollers
-      kubectl apply -f "${mfdir}/kube-dashboard-de.yaml"
-
       # Services
-      for manifest in {kube-dns,heapster,kube-dashboard}-svc.yaml;do
+      for manifest in {kube-dns,heapster,kube-dashboard}-svc.yaml; do
           kubectl apply -f "${mfdir}/$manifest"
```

Comment:
Nit: we should keep the style consistent, so you might want to move the `.yaml` suffix to `kubectl apply`.